RELEASE.md

```diff
@@ -1,3 +1,75 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Cloud TPU:
+  * Experimental tf.data integration for Keras on Cloud TPUs.
+* Add C functions for querying kernels
+* Add C++ functions for querying kernels
+* Add Python functions for querying kernels
+
+## Breaking Changes
+
+* Keras:
+  * The default values for tf.keras `RandomUniform`, `RandomNormal`, and `TruncatedNormal` initializers have been changed to match those in external Keras.
+* `tf.contrib`:
+  * Breaking change: `model.get_config()` on a Sequential model now returns a config dictionary (consistent with other Model instances) instead of a list of configs for the underlying layers.
```

Comment:
We should reference the full location of this. Is this tf.contrib.model?

Reply:
Huh, looks like this was in Keras but got put under tf.contrib. Updated in the upcoming push.